randovania/data/json_data/prime3.txt

```diff
@@ -1830,9 +2159,16 @@ Asset id: 691790544662692104
       Trivial
   > Dock to Refinery Access
       Plasma Beam
+  > Dock to Gel Purification Site
+      After Gel Refinery Site Pipe
   > Dock to Main Lift
       Trivial
 
+> Event - Pipe; Heals? False
+  * Event Gel Refinery Site Pipe
+  > Dock to Gel Purification Site
+      Trivial
+
 ----------------
 Gel Purification Site
 Asset id: 14145904616390261937
```

Comment:
For Gel Hall (below):

- Dock to Gel Processing Site should be an Ice Missile Lock. All 3 item requirements really aren't needed and can be replaced with a damage requirement. For now, put Normal Damage. A "Fuel Gel Damage" can be added later.

Reply:
I redid some of the logic of this room. Basically, everything links only to the Falling Platform, except for Gel Purification Site, which links to Save Station A

Can't jump high enough out of fuel gel to get the missile on the ledge (not sure if you can SA). So, falling platform -> ledge missile requires ice missiles, and ledge missile -> falling platform requires ice missiles or damage.